sdk/tests/fixtures/rustls/es256_private_key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
That mass of test files felt a little loud, so I put them in their own directory

Reply:
Removed